test/execflags/configFile/configFileErrorLineno.future

```diff
@@ -0,0 +1,5 @@
+feature request: line numbers for config file errors
+
+It seems like it would be nice if errors in parsing configuration
+files generated line numbers along with the filename, but this is more
+than I have time to take on today.
```

Comment:
Missing a newline at the end of the file.  Also, I think this should reference the issue you opened to track it.